requirements-dev.txt

```diff
@@ -2,3 +2,7 @@ coverage==4.4.1
 nose==1.3.7
 pylint==1.7.2
 tox==2.8.2
+twine==1.12.1
+setuptools==40.4.3
+wheel==0.32.2
+readme-renderer==23.0
```

Comment:
Are these dependencies necessary for publishing the package? Are they installed with `twine`?

Reply:
`setuptools` and `wheel` are not dependencies of twine but the tutorial linked in the orginal PR mentions that new-ish versions are required to support building the distribution using markdown.


A fresh (python 3.7) virtualenv has these already installed for me.
```
pip        18.1
setuptools 40.5.0
wheel      0.32.2
```
and install `twine` with pip 18.1, returns the following total list of installed packages
```
Package           Version
----------------- ----------
bleach            3.0.2
certifi           2018.10.15
chardet           3.0.4
docutils          0.14
idna              2.7
pip               18.1
pkginfo           1.4.2
Pygments          2.2.0
readme-renderer   24.0
requests          2.20.0
requests-toolbelt 0.8.0
setuptools        40.5.0
six               1.11.0
tqdm              4.28.1
twine             1.12.1
urllib3           1.24
webencodings      0.5.1
wheel             0.32.2
```

**So I think it's possible to only add `twine==1.12.1` to requitements-dev.txt**

There was an [issue with readme_renderer](https://github.com/pypa/twine/issues/404) which seems to have been fixed with "twine==1.12.1" so it is safe to remove `readme_renderer` from requirements-dev.text

Reply:
Ah, makes sense. Can you make the required changes before I review again?